.github/dependabot.yml

```diff
@@ -12,3 +12,8 @@ updates:
     interval: monthly
     time: "13:00"
   open-pull-requests-limit: 10
+- package-ecosystem: "github-actions"
+  directory: "/"
+  schedule:
+    interval: monthly
+    time: "13:00"
```

Comment:
Why 1pm? Arbitrary?

Reply:
Yep, this was generated by default when we added these configs for the existing package systems. Doesn't really matter.